environments/icds-cas/inventory/citusdb.csv

```diff
@@ -41,13 +41,13 @@ citusdb_master,postgresql_max_wal_size,8GB,S,,,,,,
 citusdb_master,postgresql_max_worker_processes,32,I,,,,,,
 citusdb_master,postgresql_min_wal_size,4GB,S,,,,,,
 citusdb_master,postgresql_random_page_cost,1.1,F,,,,,,
-citusdb_master,postgresql_shared_buffers,20GB,S,,,,,,
+citusdb_master,postgresql_shared_buffers,40GB,S,,,,,,
 citusdb_master,postgresql_wal_buffers,16MB,S,,,,,,
 citusdb_master,postgresql_work_mem,1872kB,S,,,,,,
 citusdb_worker,pgbouncer_default_pool,450,I,,,,,,
 citusdb_worker,postgresql_checkpoint_completion_target,0.9,F,,,,,,
 citusdb_worker,postgresql_default_statistics_target,500,I,,,,,,
-citusdb_worker,postgresql_effective_cache_size,60GB,S,,,,,,
+citusdb_worker,postgresql_effective_cache_size,80GB,S,,,,,,
```

Comment:
I think this should have been on `citusdb_master`

Reply:
Yep it should be

Reply:
Fixed